docs/getting-started.rst

```diff
@@ -17,10 +16,226 @@ Creating a New Project
 Adding a New App
 ~~~~~~~~~~~~~~~~
 
+Defining models
+~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Add a few words with a link to Django models docs

Reply:
done